src/main/java/io/vertx/core/http/impl/HttpClientImpl.java

```diff
@@ -646,13 +646,4 @@ private void checkClosed() {
       throw new IllegalStateException("Client is closed");
     }
   }
-
-  @Override
-  protected void finalize() throws Throwable {
```

Comment:
I agree :)

Reply:
This one just calls NetClientImpl.close(), and it has its own finalizer.